docs/troubleshooting-guide/sandboxing.md

```diff
@@ -0,0 +1,76 @@
+
+## Troubleshooting Process
+
+### Users can see data they're not supposed to be able to see
+
+**How to detect this:** People can view data that they shouldn't be able to.
+
+**How to fix this:** The fix depends on the root cause.
+
+1. Sandboxing isn't implemented for non-SQL databases like MongoDB, Druid, or Google Analytics.
+
+2. Sandboxing isn't implemented for SQL questions: any user with permissions to view the question can see all the results.
+
+3. Public questions can't be sandboxed: if someone doesn't have to log in to view the question, Metabase doesn't have user properties or group properties available for filtering the data, so all results will be shown. (We recommend that you do *not* using public sharing when you are sandboxing data.)
+
+4. If the administrator didn't restrict access to the underlying table when setting up sandboxing, people will be able to see the original data. You can check this by going into the Admin Panel and viewing Permissions for the table in question.
+
+5. [Signed embedding][signed-embedding] will show all results by default, though it's possible to control filtering with [locked parameters][locked-parameters]. Note that signed embedding depends only on the token generated by the including page, not whether someone is logged into Metabase or not.
+
+6. If users are logging in with single sign-on but the expected attributes aren't being saved and made available, sandboxing will deny access. Our article on [Authenticating with SAML][authenticating-with-saml] explains the required setup in detail.
+
+### Users can't see the data they're supposed to be able to see
+
+**How to detect this:** Someone is supposed to be able to use some of the values in a table in their queries, but are denied access or get an empty set of results where there should be data.
+
+**How to fix this:** Again, the fix depends on the root cause:
+
+1. Administrators usually restrict access to tables as part of sandboxing. If the restrictions are too tight by mistake (e.g., "no access") then people might not be able to see any data at all. You can check this by going into the Admin Panel and viewing Permissions for the table in question.
+
+2. Some fields in a table which the user *does* have sandbox access to are using remapping to display information from another table which the user does *not* have sandbox access to. You can check this by going into the Admin Panel and viewing the Data Model for the fields in question.
+
+### A user is in several groups but can't see the sandboxed data
+
+We only allow [one sandbox per table][one-sandbox-per-table]: if someone is a member of two or more groups with different permissions, every rule for figuring out whether access should be allowed or not is confusing. We therefore only allow one rule, which sometimes means the administrator will create a new group to capture precisely who's allowed access to what.
```

Comment:
Should we rewrite to follow the how to detect this, how to fix this template? With the fix here being to create a new group?

Reply:
Done.